src/stanrun/cmdline.jl

```diff
@@ -52,7 +52,7 @@ function cmdline(m::OptimizeModel, id)
             if m.algorithm == :lbfgs
                 cmd = `$cmd history_size=$(m.history_size)`
             end
-        elseif m.algorithn == :newtom
```

Comment:
Is this double typo? Should this be :newton?
https://mc-stan.org/cmdstanr/reference/model-method-optimize.html

Reply:
Done :)